ark/phenotyping/cluster.py

```diff
@@ -46,6 +46,58 @@ def winner(sample, weights):
     return winning_coords
 
 
+def batch_winner(samples, weights):
+    """Predict multiple samples at once
+
+    Args:
+        samples (numpy.ndarray):
+            Contains the rows of the pixel matrix to predict
+        weights (numpy.ndarray):
+            A weight matrix of dimensions [num_x, num_y, num_chans]
+
+    Returns:
+        list:
+            The list with indices corresponding to the prediction of the respective sample
+    """
+
+    # only 1 sample provided: expand to 2-D
+    if len(samples.shape) == 1:
+        samples = np.expand_dims(samples, axis=0)
```

Comment:
when would we just have one sample?

Reply:
This would happen in `cluster_som` if the batches divided up unequally with 1 row remaining.

Reply:
gotcha.  Either way, couldn't we just call/return the regular `winner` function in this case?

Reply:
Yeah, good point. I did it this way because I wasn't sure if we'd keep `winner` at all. Now that we're potentially moving forward with a different method, could be a moot point.